homeassistant/components/sensor/youless.py

```diff
@@ -0,0 +1,119 @@
+"""
+@ Author      : Gerben Jongerius
+@ Date        : 04/29/2018
+@ Description : Youless Sensor - Monitor power consumption. This component will add the following sensors
+                   - Current power consumption (in W)
+                   - Current tick count, since the Youless meter started running (in W)
+"""
+import json
+import logging
+from datetime import timedelta
+from urllib.request import urlopen
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_MONITORED_VARIABLES
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
```

Comment:
Please keep the imports grouped.

Reply:
I've updated the imports to be grouped by source. I hope that is what you meant with this.